python/docs/requirements.txt

```diff
@@ -0,0 +1,4 @@
+sphinx
+sphinxcontrib
```

Comment:
right. updated. can you let me know if that gets the docs building on your machine?